.gitlab-ci.yml

```diff
@@ -47,7 +47,7 @@ test-ef-fake-crypto:
     GIT_SUBMODULE_STRATEGY: normal
   script:
     - make make-ef-tests
-    - cargo test --manifest-path tests/ef_tests/Cargo.toml --release --features ef_tests fake_crypto
+    - cargo test --manifest-path tests/ef_tests/Cargo.toml --release --features ef_tests,fake_crypto
```

Comment:
Cargo 1.39 changed how `--features A B` is interpreted, apparently to fix a regression... which we had come to rely on.